pymilo/chains/clustering_chain.py

```diff
@@ -92,20 +95,26 @@ def serialize_clusterer(clusterer_object):
     return clusterer_object.__dict__
 
 
-def deserialize_clusterer(clusterer):
+def deserialize_clusterer(clusterer, is_inner_model=False):
```

Comment:
`is_inner_model` docstring

Reply:
you're right, sure.
I will also add a section in supported models to introduce supported preprocessing modules.
@sepandhaghighi

Reply:
Applied
ready for your review
@sepandhaghighi